.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The whole .idea directory should be ignored because is a folder that intellij uses and is not relevant to your code.
Please create a new .gitignore file using this tool https://www.toptal.com/developers/gitignore and type intellij to create it. It will also make all de *.class file to be ignored as well.

![image](https://user-images.githubusercontent.com/30251958/115129146-f180bc00-9fa8-11eb-9ef3-760d84a322d5.png)

Reply:
Fixed.